documents/how_does_the_cli_work.md

```diff
@@ -90,8 +90,18 @@ The data parameters are then passed to the DataLoader class _**dataloader.ts**_.
 The DataLoader loads the data depending on the file and path type, i.e., single file, multiple files, or a directory.
 The simulation and shading geometries can be loaded either from JSON in the format _{ positions: number[] }_ or from an OBJ file.
 For irradiance data, a JSON file in the format of _SolarIrradianceData_ is required. The smallest possible parameterization
-for a run is _**--simulation-geometry**_ and _**--irradiance-data**_. After the data has been loaded, it is transferred
-to the headless Chrome runner _**headlessBrowser.ts**_ together with the CLI parameters.
+for a run is _**--simulation-geometry**_ and _**--irradiance-data**_. 
+
+Before the data gets moved into the browser memory, 
+the _**--shading-geometry**_ gets filtered. The value of _**--min-sun-angle**_ is used if it is set, otherwise the minimum
```

Comment:
Can you add a sentence why it is filtered: "Objects that are far away do not shade our simulation geometry, hence they can be negelcted" or something like this.